result/src/commonMain/kotlin/com/github/kittinunf/result/Result.kt

```diff
@@ -3,6 +3,10 @@ package com.github.kittinunf.result
 import com.github.kittinunf.result.Kind.Failure
 import com.github.kittinunf.result.Kind.Success
 
+fun<V: Any> V.success(): Result.Success<V> = Result.success(this)
```

Comment:
It looks like we should have a single space between `fun` and `<`, can you please format this for me 🙏 ?

Reply:
:done: 
1a795fa63313fceeae910b9d2be2276ecba1c149